package.json

```diff
@@ -170,7 +170,7 @@
     "ethereumjs-wallet": "1.0.1",
     "events": "3.3.0",
     "expo": "53.0.11",
-    "expo-image-picker": "16.1.4",
+    "expo-image-picker": "17.0.0-canary-20250713-8f814f8",
```

Comment:
Are you ok with canary version? It improved dark mode support and the picker looks a lot better.